test/formatters.test.js

```diff
@@ -4,11 +4,20 @@
 const { hostname } = require('node:os')
 const { join } = require('node:path')
 const { readFile } = require('node:fs').promises
-const { test } = require('tap')
+const { test } = require('node:test')
 const { sink, once, watchFileCreated, file } = require('./helper')
 const pino = require('../')
 
-test('level formatter', async ({ match }) => {
+function match (t, obj, expected) {
```

Comment:
Subsequent to the other PR being updated with `@jsumners/assert-match` and being merged, this PR needs to be updated to use that assertion function.

Reply:
yep, done!